encoding/protobuf.go

```diff
@@ -106,6 +108,50 @@ func (p protoSerializer) Encoding() Encoding {
 	return Protobuf
 }
 
+func (p protoSerializer) ErrorDetails(err error) ([]interface{}, error) {
+	// Here we use yarpcerrors since the transport layer of yab is using yarpc as well
+	if err == nil || !yarpcerrors.IsStatus(err) {
```

Comment:
no need to check `err == nil`: 
from [IsStatus doc](https://pkg.go.dev/go.uber.org/yarpc@v1.53.2/yarpcerrors#IsStatus): This is false if the error is nil.